incubator/hnc/Makefile

```diff
@@ -19,6 +19,15 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Set default version tag for krew build (unless version is set)
+ifeq (,${VERSION})
+VERSION=v0.1.0
+endif
+
+# Get check sum value of krew archive
+KREW_CKSM=$(shell sha256sum hack/hnc.tar.gz | cut -d " " -f 1)
```

Comment:
I would have thought that this got evaluated once when the makefile is first loaded, which means that if you call `make krew-build` and the archive is _modified_, then this would return the sha256 of the _old_ version, not the new one. You could test this by deleting `hack/hnc.tar.gz` (or as I suggest below, `bin/kubectl-hnc.tar.gz`) and seeing what this variable evaluates to - I would _guess_ it's an empty string, even if it gets built while `make` is running.

Reply:
tested and work